packages/core/components/Drawer/index.tsx

```diff
@@ -22,35 +22,42 @@ const DrawerDraggable = ({
   children,
   id,
   index,
+  canInsert,
 }: {
   children: ReactNode;
   id: string;
   index: number;
-}) => (
-  <Draggable
-    key={id}
-    id={id}
-    index={index}
-    showShadow
-    disableAnimations
-    className={() => getClassNameItem()}
-  >
-    {() => children}
-  </Draggable>
-);
+  canInsert?: boolean;
+}) => {
+  return (
+    <Draggable
+      key={id}
+      id={id}
+      index={index}
+      isDragDisabled={!canInsert}
+      showShadow
+      disableAnimations
+      className={() => getClassNameItem()}
+    >
+      {() => children}
+    </Draggable>
+  );
+};
 
 const DrawerItem = ({
   name,
   children,
   id,
   label,
   index,
+  canInsert,
```

Comment:
I think we should call this `isDragDisabled` rather than `canInsert`, as this is (sort-of) designed to be used in other contexts that aren't necessarily about _inserting_ elements.